CMakeLists.txt

```diff
@@ -19,12 +19,11 @@ if (UNIX)
       execute_process(COMMAND xcrun --show-sdk-path
                       OUTPUT_VARIABLE CMAKE_OSX_SYSROOT
                       OUTPUT_STRIP_TRAILING_WHITESPACE)
-      if (LLVM_DIR STREQUAL "" OR NOT LLVM_DIR)
-         set(LLVM_DIR "/usr/local/Cellar/llvm@4/4.0.1/lib/cmake/llvm")
-      endif()
    endif()
 endif()
 
+set(LLVM_DIR "$ENV{HOME}/opt/llvm/lib/cmake/llvm")
```

Comment:
Does this break users who do not use the script? If a path is unique to how our build script sets things up it should probably be set from the build script that sets things up

Reply:
We could just remove this entirely, right? It was only for Darwin originally, and never set for Linux, but from what I can tell if LLVM_DIR isn't set it looks under what I've specified in the ENV vars we have users set.

Reply:
Yeah it looks like you can remove it since you export `LLVM_DIR` from the script it will be picked up that way

Reply:
Ah, actually, we need to keep it. Most of the folks here don't use the build scripts to build, so they won't get the ENV I set. I'll expand this for BOOST_ROOT as well and interpolate in the EOSTester files

Reply:
But does it matter? At least on macos and ubuntu I don't think you need anything at all here right?

Reply:
I'm not sure actually. Bucky was saying they should both be there. I've wrapped them with if statements to avoid setting them if they already exist.

Reply:
AFAIK llvm4 on brew & ubuntu should be installed in such a way that this isn't needed since the cmake module is installed in a system location. haven't tried myself, but we should be more sure of our changes

Reply:
ah okay for macos we do need to specify where LLVM is, it won't get added to system cmake directory